### PR TITLE
fix: resolve patched @xmldom/xmldom

### PR DESCRIPTION
**Pull request by estrattonbailey**
## What changed (plus any additional context for devs)
Only a `0.0.2` patch increment. Treats https://github.com/advisories/GHSA-crh6-fp67-6883

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -371,7 +371,6 @@
     "**/resolve": "1.8.1",
     "**/semver": "7.3.2",
     "**/terser": "5.14.2",
-    "**/xmldom": "github:xmldom/xmldom#c568938641cc1f121cef5b4df80fcfda1e489b6e",
```
Comment (estrattonbailey) [inline package.json:374]: Nothing depends on this library. Not sure when that changed, but we don't need this.

**Review by skylarbarrera — APPROVED** [review]
send it

**Comment by estrattonbailey** [comment]
@SocketSecurity ignore es5-ext@0.10.53

**Comment by estrattonbailey** [comment]
Ignoring this because it [appears to be harmless](https://github.com/medikoo/es5-ext/issues/186), and we don't run package.json scripts anyway via our `ignore-scripts` yarnrc config.

**Pull request merged**